concise-mid-tag.cddl

```diff
@@ -174,28 +174,17 @@ tagged-svn = #6.552(svn)
 tagged-min-svn = #6.553(min-svn)
 svn-type-choice = tagged-svn / tagged-min-svn
 
-flags-type = [
```

Comment:
So we are moving from bitmask to map with labels ("code points") here. Is this a general trend or why would we be doing that only here?

Reply:
> So we are moving from bitmask to map with labels ("code points") here. Is this a general trend or why would we be doing that only here?

I don't see any other places in the CDDL where we use bit-masks (including imported modules, i.e., CoSWID):

```
╭─thofos01 at tho in ~/Documents/github.com/ietf-rats/ietf-corim-cddl on new-flags-encoding✘✘✘
╰─± grep -c '.bits' *.cddl
comid-autogen.cddl:0
comid-code-points.cddl:0
common.cddl:0
concise-mid-tag.cddl:0
concise-rim.cddl:0
concise-swid-tag.cddl:0
corim-autogen.cddl:0
corim-code-points.cddl:0
corim.cddl:0
flags.cddl:0
generic-non-empty.cddl:0
signed-corim.cddl:0
xcorim-autogen.cddl:0
xcorim-code-points.cddl:0
xcorim.cddl:0
```

So, apparently `flags-type` was the exception we got rid of :-)

Reply:
oh. maybe I should grep before I ask for an obvious answer. Then I agree with this change!